tests/conftest.py

```diff
@@ -50,6 +50,13 @@
     import pytz
 
 
+# Don't collect `test_official.py` on Python 3.10- since it uses newer features like X | Y syntax.
+# Docs: https://docs.pytest.org/en/7.1.x/example/pythoncollection.html#customizing-test-collection
+collect_ignore = []
+if sys.version_info < (3, 10):
+    collect_ignore.append("test_official.py")
```

Comment:
Nice addition 👍 Is it possible (without too much effort) to issue a warning in case someone has set the TEST_OFFICIAL flag, but is running on 3.10-?

Reply:
should be possible

Reply:
ok nvm, pytest doesn't actually show any output unless you use the `-s` flag. Tried using `warnings.warn`, print() and `logging.info`

Reply:
shame. But okay, doesn't matter much :) Thanks for checking this out!